backend/src/v5/utils/helper/yup.js

```diff
@@ -147,7 +149,10 @@ YupHelper.types.embeddedImage = (isNullable) => imageValidityTests(
 YupHelper.types.embeddedImageOrRef = () => imageValidityTests(
 	Yup.mixed()
 		.transform((currValue, orgVal) => {
-			if (orgVal) return isUUIDString(orgVal) ? stringToUUID(orgVal) : Buffer.from(orgVal, 'base64');
+			if (orgVal && typeof orgVal === 'string') {
```

Comment:
the reason is because we are running the validation twice here 


so the first time is converting a UUID string to UUID but the second time is trying to convert a UUID to buffer. The only other place we are using embeddedImageOrRef is comments but we are only validating them once so we dont have the same problem

Reply:
ok fair enough. in which case can we use isString from typechecker?